src/clr-angular/utils/drag-and-drop/interfaces/drag-event.ts

```diff
@@ -7,14 +7,18 @@ export enum ClrDragEventType {
     DRAG_START,
     DRAG_MOVE,
     DRAG_END,
+    DRAG_ENTER,
+    DRAG_LEAVE,
     DROP
 }
 
-export interface ClrDragEvent<T> {
-    // TODO: this interface will be expanded and customized as the implementation progresses.
+export interface ClrDragEventInternal<T> {
```

Comment:
If this is the internal event now, then we shouldn't prefix it right? Just `DragEvent` or `DragEventInternal`